libs/remix-tests/src/testRunner.ts

```diff
@@ -377,6 +377,11 @@ export function runTest (testName: string, testObject: any, contractDetails: Com
           context: testName,
           web3
         }
+        if (err.message.includes('Transaction has been reverted by the EVM')) {
```

Comment:
even if the transaction failed, you should be able to get a receipt.. it looks like this will break as soon as the error label change. but does seem we can do better..

Reply:
Yes, according to current web3 implementation, we don't get tx hash on error